config.yaml

```diff
@@ -142,3 +142,72 @@ options:
       (no floating IP) by default.
     type: boolean
     default: false
+  nrpe-network-ids:
+    description: |
+      List of OpenStack networks IDs to be checked by NRPE check. All values
+      must be separated by a comma or 'all' can be entered.
+    type: string
+    default: "all"
+  nrpe-skip-network-ids:
+    description: |
+      List of OpenStack networks IDs to be skipped by the NRPE check.
+      All values must be separated by a comma. The condition for using this
+      option is to set "nrpe-network-ids" to "all" or this value will raise
+      an error.
+    type: string
+    default: ""
+  nrpe-subnet-ids:
+    description: |
+      List of OpenStack subnets IDs to be checked by NRPE check. All values
+      must be separated by a comma and the value `all` is not allowed.
+    type: string
+    default: ""
+  nrpe-port-ids:
+    description: |
+      List of OpenStack ports IDs to be checked by NRPE check. All values
+      must be separated by a comma or 'all' can be entered.
+    type: string
+    default: "all"
+  nrpe-skip-port-ids:
+    description: |
+      List of OpenStack ports IDs to be skipped by the NRPE check.
+      All values must be separated by a comma. The condition for using this
+      option is to set "nrpe-port-ids" to "all" or this value will raise
+      an error.
+    type: string
+    default: ""
+  nrpe-floating-ip-ids:
+    description: |
+      List of OpenStack floating ips IDs to be checked by NRPE check. All
+      values must be separated by a comma or 'all' can be entered.
+    type: string
+    default: "all"
+  nrpe-skip-floating-ip-ids:
+    description: |
+      List of OpenStack floating ips IDs to be skipped by the NRPE check.
+      All values must be separated by a comma. The condition for using
+      this option is to set "nrpe-floating-ip-ids" to "all" or this value
+      will raise an error.
+    type: string
+    default: ""
+  nrpe-server-ids:
+    description: |
+      List of OpenStack servers IDs to be checked by NRPE check. All values
+      must be separated by a comma or 'all' can be entered.
+    type: string
+    default: "all"
+  nrpe-skip-server-ids:
+    description: |
+      List of OpenStack servers IDs to be skipped by the NRPE check.
+      All values must be separated by a comma. The condition for using this
+      option is to set "nrpe-server-ids" to "all" or this value will raise
+      an error.
+    type: string
+    default: ""
```

Comment:
i noticed the pattern here for nrpe-<resource>-id and nrpe-skip-<resource>-id, but there's no config for load-balancer ids .  Can you add the same ability for load-balancers?